spec/twilio_txt_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'twilio_txt'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.